test/test_overrides.py

```diff
@@ -29,6 +31,11 @@
 
 Tensor = torch.Tensor
 
+if os.getenv("ATEN_CPU_CAPABILITY") in ("default", "avx2"):
+    # This test is not supported on ARM
+    print("Skipping due to failing when cuda build runs on non cuda machine, see #150059 for example")
+    sys.exit()
```

Comment:
Have you tried using `pytest.skip(allow_module_level=True)` ([docs](https://docs.pytest.org/en/stable/reference/reference.html#pytest-skip))?  That might let you properly skip all tests (and have them be marked as skipped) instead of this silent skip

Reply:
I just tried this on a dummy test file, unfortunately it just says 1 skipped for the entire file and I can't seem to get it to print the skip reason either